src/main/webapp/scripts/jasmine/SpecRunner.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Jasmine Spec Runner v3.6.0</title>
+
+  <link rel="shortcut icon" type="image/png" href="lib/jasmine-3.6.0/jasmine_favicon.png">
+  <link rel="stylesheet" href="lib/jasmine-3.6.0/jasmine.css">
+
+  <script src="lib/jasmine-3.6.0/jasmine.js"></script>
+  <script src="lib/jasmine-3.6.0/jasmine-html.js"></script>
+  <script src="lib/jasmine-3.6.0/boot.js"></script>
```

Comment:
Instead of storing third party libraries/frameworks to your local server, you can use **CDN**s.
There are multiple **advantages** of using those:
1) It is **beneficial for users**. A CDN usually has a distributed, fast and stable architecture, so the user will receive a cached copy of the file from a physical location near them -> **Faster loading time**
2) It is easier for us to see what needs to be reviewed.
3) Our repo size is much **smaller**, so cloning/switching branches (or other actions) will be **faster**. 

You can add this and probably you won't need "jasmine" directory:
```javascript
<script src="https://cdnjs.cloudflare.com/ajax/libs/jasmine/3.6.0/jasmine.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/jasmine/3.6.0/jasmine-html.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/jasmine/3.6.0/boot.min.js"></script>
```

Reply:
Also at the beginning when such optimization may not even be as important, inclusion via CDN can just be more convenient. That's at least my #1 reason to start off with that.